### PR TITLE
No unlock ring

### PR DESCRIPTION
**Pull request by Boruch-Baum**
This addresses upstream (master) issue #120 by @davidv171

### CODE REVIEW

File: i3lock-fancy [hunk 1]
```diff
@@ -77,6 +79,7 @@ while true ; do
 	    convert -list font | awk -F: '/Font: / { print $2 }' | sort -du | command -- ${PAGER:-less}
 	    exit 0 ;;
 	-n|--nofork) i3lock_cmd+=(--nofork) ; shift ;;
+        -u|--nounlockindicator) i3lock_cmd=(i3lock -i "$image" -u) ; shift ;;
```
Comment (AladW) [inline i3lock-fancy]: Why are you resetting the command instead of `i3lock_cmd+=(-u)`?

Comment (Boruch-Baum) [reply i3lock-fancy]: Why *not*? The only differences seem to me to be improved readability, and avoiding repetitive options if some user script ends up passing "-u -u -u -u".

Comment (meskarune) [reply i3lock-fancy]: It's possible someone can have flags which are then over written, such as:
    i3lock-fancy -gp -f Noto-Sans -u

Comment (Boruch-Baum) [reply i3lock-fancy]: @meskarune : All the options that you mentioned are handled differently by the script, so nothing would be overwritten. Wait a minute, the no-fork option does conflict, so I do need to make the change.

**Comment by AladW** [comment]
Looks like there's a conflict with https://github.com/meskarune/i3lock-fancy/commit/b77540eb2bc458c85410b6fcb26a68ab9e1d6432

**Comment by Boruch-Baum** [comment]
@AladW  

1] At this point, after the delay and intervening commits, why are you surprised?

2] Did you bother to look at b77540e and see how trivial that issue is?

3] Your pettiness is tiring.

4] At this point, our versions have diverged so much that any further offers to contribute here will be difficult to do via pull requests at all; I'll still share via posting issues and referencing code snippets, unless I give up on your personality. At this point, my version of the script does more, fills the requests of users who have asked for new features, and is more computationally efficient.

**Comment by AladW** [comment]
You have some issues buddy. Hope you figure them out.

**Pull request closed without merging**